### DISCUSSION TITLE
Community code reviews

### DISCUSSION THREAD

**Discussion by ericzbeard** [aws-cloudformation/rain#240] [General]
Would anyone be interested in helping out with code reviews on new features for rain? Sometimes we get a new feature submitted and I wouldn't mind a second opinion from someone in the community. And I'm working on some experimental features mostly in isolation (like [this PR](https://github.com/aws-cloudformation/rain/pull/239) for adding intrinsic support to `rain cc deploy`).

Pinging a few recent contributors, thanks so much for your PRs and feedback!

@hariprakash-j
@iainelder
@null93
@b0tting
@jsutton 
@vjeantet 
@sho-saito 
@bartleboeuf 
@khmoryz 

I'm not asking for an always-available commitment here, just anyone who wouldn't mind getting tagged as an optional reviewer occasionally.

**Comment by hariprakash-j** [comment]
hi @ericzbeard , I won't mind helping out with code reviews : )

**Comment by ericzbeard** [comment]
I'm working on the approvals I need to add you as a collaborator so you can be marked as a reviewer on PRs (otherwise we would just need to rely on comments/reactions)

**Comment by khmoryz** [comment]
@ericzbeard 
I'd like to participate in code reviews when I have time😀

**Reply by ericzbeard** [reply]
Cool, I'll ping you on future PRs - no obligation, but will be great to have the feedback.

**Comment by ericzbeard** [comment]
@null93 are you also interested? I see the thumbs up.

**Reply by null93** [reply]
Count me in!

**Comment by iainelder** [comment]
Hi, @ericzbeard , glad to see Rain keeps getting better!

Yes, I'd be glad to help out when I can.

I don't have much experience in Go yet, but I this seems like a good opportunity to learn.

Even if I don't understand the code, I can try out new builds give feedback.

**Comment by sho-saito** [comment]
Hi, @ericzbeard 

I have recently been very helped by this tool in work.
So I feel that I would like to contribute something, but unfortunately I have very little experience with Go.
But it is a good opportunity to learn. So I hope to be able to contribute by giving feedback or something in the beginning if possible!

**Reply by ericzbeard** [reply]
Good to hear Rain is helping you! Happy to tag you for feedback when I develop new features. In the meantime, you could go through the open issues list and add reactions/comments to the ones that sound interesting. Thanks!
